toss/trajectory_tools.py

```diff
@@ -70,13 +130,13 @@ def compute_trajectory(x: np.ndarray, args, func: Callable) -> Union[np.ndarray,
     return trajectory_info, squared_altitudes, collision_detected
 
 
-def integrate(func: Callable, x: np.ndarray, args):
+def integrate_trajectory(func: Callable, x: np.ndarray, args):
```

Comment:
I am not sure this renaming is helpful. Was is to avoid some name conflict? Then I would rather go for integrate_system or something. because a trajectory is not really something you can integrate over? :thinking:

Reply:
Fixed.